inc/class-main.php

```diff
@@ -41,6 +41,18 @@ public function init() {
 			add_filter( 'upload_mimes', array( $this, 'allow_meme_types' ) ); // phpcs:ignore WordPressVIPMinimum.Hooks.RestrictedHooks.upload_mimes
 			add_filter( 'wp_check_filetype_and_ext', array( $this, 'fix_mime_type_json_svg' ), 75, 4 );
 		}
+
+		add_filter(
+			'themeisle_sdk_compatibilities/' . OTTER_BLOCKS_BASEFILE,
```

Comment:
You need to use `basename( OTTER_BLOCKS_PATH )` instead of OTTER_BLOCKS_BASEFILE.